### PR TITLE
Awesome books: Add ES6 modules

### PR DESCRIPTION
**Pull request by ernestmusong**
## I have refactored my Awesome books to use ES6 modules as follows:

- I have split javascript files in to modules
- I have imported these modules into the index.js file which is the entry file.
- I have implemented a dynamic time using [luxon ](https://moment.github.io/luxon/#/install?id=es6) as dependency.
- I have maintained my html and css files

### CODE REVIEW

File: modules/storage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
```
Comment (ABDULALI3468) [inline modules/storage.js:33]: Please consider reviewing your remove books code as they are not updating the local storage and due to that reason it deleted the book when we click the remove button but when we load the page it still persists there. Consider reviewing that.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @ernestmusong , 



Indeed you did an exceptional job but still, there is a small problem that needs to be addressed to finish this project.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/storage.js (hunk 1 above)
Comment (ernestmusong) [reply modules/storage.js:33]: > Please consider reviewing your remove books code as they are not updating the local storage and due to that reason it deleted the book when we click the remove button but when we load the page it still persists there. Consider reviewing that.

Hi @ABDULALI3468 , a big thank you for your feedback. However I have fixed the issue rather in [ui.js](https://github.com/ernestmusong/Awesome-books-ES6/blob/Add-ES6-modules/modules/ui.js#:~:text=%7D-,static%20deleteBook(elem%2C%20id)%20%7B,%7D,-static%20clearFields) and in [index.js](https://github.com/ernestmusong/Awesome-books-ES6/blob/Add-ES6-modules/index.js#:~:text=document.querySelector(%27%23container,%7D)%3B). It works now.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ernestmusong, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**